### DISCUSSION TITLE
Images in readme are not resizing on the mobile app

### DISCUSSION THREAD

**Discussion by madhusudanbabar** [community/community#14] [Mobile]
When i visit any repo from phone browser, it works well but when i open the same repository in the android app, the images looks to big

Here's are the screenshots: 
- from mobile app
![mobile app](https://user-images.githubusercontent.com/28552728/96333532-2195e380-1088-11eb-9b94-068c9b427a69.png)

- from the website 
![website](https://user-images.githubusercontent.com/28552728/96333564-56099f80-1088-11eb-864f-9d4648279f56.png)

**Comment by alcere** [comment]
@madhusudanbabar thanks for reporting! We are sorry for the troubles.
Can you please tell us which version of the app are you running?
Thanks!

**Reply by madhusudanbabar** [reply]
Android application V1.3.1

**Reply by alcere** [reply]
Thanks! We were able to reproduce it, the team is already working on the fix ;)

**Comment by alcere** [comment]
We are truly sorry.
After some initial investigations we found that images using width and height attributes of the img HTML tag were actually not supported on mobile at the moment. We will monitor this space and make potential improvements when we have more bandwidth in the future.

**Reply by Timmoth** [reply]
Did any improvements get made on this?
